components/list/ListEntry.vue

```diff
@@ -83,11 +104,27 @@ async function removeList() {
     deleteBusy = false
   }
 }
+
+function clearError() {
+  actionError = undefined
+  nextTick(() => {
+    if (isEditing)
+      input.value?.focus()
+    else
+      deleteBtn.value?.focus()
+  })
+}
+
 onBeforeUnmount(() => cancelEdit(false))
```

Comment:
we should change this to `onDeactivated`, the paginator still there on page transition (app with keep alive)